tests/integration-all/websocket/tests.js

```diff
@@ -43,12 +43,54 @@ describe('AWS - API Gateway Websocket Integration Test', function() {
     return removeService(tmpDirPath);
   });
 
+  async function getWebSocketServerUrl() {
+    const result = await awsRequest('CloudFormation', 'describeStacks', { StackName: stackName });
+    const webSocketServerUrl = _.find(result.Stacks[0].Outputs, {
+      OutputKey: 'ServiceEndpointWebsocket',
+    }).OutputValue;
+
+    return webSocketServerUrl;
+  }
+
+  describe('Two-Way Setup', () => {
+    it('should expose a websocket route that can reply to a message', async () => {
+      const webSocketServerUrl = await getWebSocketServerUrl();
+
+      return new Promise((resolve, reject) => {
+        const ws = new WebSocket(webSocketServerUrl);
+        reject = (promiseReject => error => {
+          promiseReject(error);
+          try {
+            ws.close();
+          } catch (closeError) {
+            // safe to ignore
+          }
+        })(reject);
+
+        ws.on('error', reject);
+        ws.on('open', () => {
+          console.info("Sending message to 'hello' route");
+          ws.send(JSON.stringify({ action: 'hello', name: 'serverless' }));
+        });
+
+        ws.on('close', resolve);
+
+        ws.on('message', event => {
+          try {
+            console.info('Received WebSocket message', event);
```

Comment:
Let's use `log` library as here: https://github.com/serverless/serverless/blob/1793cf8d7a55b85fc6505ae493dcca2292e443d2/tests/integration-all/api-gateway/tests.js#L188

Then log is output only when with adequate log level setting, or when tests fail, which makes it more DX friendly

Here configured log, could look as:

```javascript
log.debug('Received WebSocket message %o', event)
```